src/Cli/dotnet/commands/dotnet-workload/config/WorkloadConfigCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Deployment.DotNet.Releases;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Workloads.Workload;
+using Microsoft.DotNet.Workloads.Workload.Install;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+
+#nullable enable
+
+namespace Microsoft.DotNet.Workloads.Workload.Config
+{
+    internal class WorkloadConfigCommand : WorkloadCommandBase
+    {
+        bool _hasUpdateMode;
+        string? _updateMode;
+        readonly IWorkloadResolverFactory _workloadResolverFactory;
+
+        private string? _dotnetPath;
+        private string _userProfileDir;
+        private readonly IWorkloadResolver _workloadResolver;
+        private readonly ReleaseVersion _sdkVersion;
+        private readonly SdkFeatureBand _sdkFeatureBand;
+
+        readonly IInstaller _workloadInstaller;
+
+        public WorkloadConfigCommand(
+            ParseResult parseResult,
+            IReporter? reporter = null,
+            IWorkloadResolverFactory? workloadResolverFactory = null
+        ) : base(parseResult, CommonOptions.HiddenVerbosityOption, reporter)
+        {
+            //  TODO: Is it possible to check the order of the options?  This would allow us to print the values out in the same order they are specified on the command line
```

Comment:
You can only get the list of tokens via the `Tokens` property. The list *should* be in the order in which they are read. But, that doesn't have a relationship with the `CliArguments` in any way, unfortunately. You'd have to make custom logic to figure that out.